MeshLib/MeshSearch/MeshElementGrid.h

```diff
@@ -72,6 +72,8 @@ class MeshElementGrid final {
 		return elements_vec;
 	}
 
+	MathLib::Point3d const& getGridMinEdgePoint() const;
```

Comment:
I'm not sure what "GridMinEdgePoint" means. can you add documentation or rename it?